### PR TITLE
[*.hu.i18n] Improved Hungarian translations

### PR DESCRIPTION
**Pull request by lapingenieur**
2nd try with more modifications...
Also changed "monoton**ic**" to "mononton**e**" in apps/code/catalog.fr.i18n (think it's a spelling mistake)

### CODE REVIEW

**Comment by lapingenieur** [comment]
je sais d'ou vient l'erreur je vais la corriger tout de suite :)

**Comment by lapingenieur** [comment]
The `ε` character seems to have some problems...

**Comment by quentinguidee** [comment]
> The `ε` character seems to have some problems...

The characters you can use are listed here: https://github.com/Omega-Numworks/Omega/blob/omega-master/kandinsky/fonts/code_points.h

**Comment by lapingenieur** [comment]
I don't understand why this test (Metrics / binary-size (pull_request)) fails...

**Comment by RedGl0w** [comment]
> I don't understand why this test (Metrics / binary-size (pull_request)) fails...

See https://github.com/numworks/epsilon/pull/1701

File: apps/code/base.hu.i18n [hunk 1]
```diff
@@ -1,15 +1,15 @@
-AddScript = "Script hozzadáadása"
+AddScript = "Script hozzadás"
```
Comment (Gegenter) [inline apps/code/base.hu.i18n:1]:
```suggestion
AddScript = "Script hozzadása"

```

File: apps/code/base.hu.i18n [hunk 2]
```diff
@@ -1,15 +1,15 @@
-AddScript = "Script hozzadáadása"
+AddScript = "Script hozzadás"
 AllowedCharactersaz09 = "Engedélyezett karakterek: a-z, 0-9, _"
-Autocomplete = "Autocomplete"
+Autocomplete = "Önkiegészítés"
 AutoImportScript = "Script automata importálása"
-BuiltinsAndKeywords = "Builtins and keywords"
+BuiltinsAndKeywords = "Beépítet fonkciók és szókincs"
```
Comment (Gegenter) [inline apps/code/base.hu.i18n:5]:
```suggestion
BuiltinsAndKeywords = "Beépítet funkciók és szókincs"

```

File: apps/code/base.hu.i18n [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 FunctionsAndVariables = "Függvények és változók"
-ImportedModulesAndScripts = "Imported modules and scripts"
-NoWordAvailableHere = "No word available here."
-ScriptInProgress = "Script in progress"
+ImportedModulesAndScripts = "Importált scriptek és modulok"
+NoWordAvailableHere = "Nincs rendelkezésre álló szó."
+ScriptInProgress = "Script müködésben"
 ScriptOptions = "Script beállítások"
-ScriptSize = "Script size"
+ScriptSize = "Script hossza"
```
Comment (Gegenter) [inline apps/code/base.hu.i18n:15]:
```suggestion
ScriptSize = "Script mérete"

```

File: apps/code/catalog.hu.i18n [hunk 4]
```diff
@@ -36,187 +36,187 @@ PythonColorOrange = "Narancssárga szín"
...
+PythonText = "(x,y) nél egy szöveget irni"
+PythonTimeFunction = "time funkció elötag"
+PythonTrunc = "Egész csonka (?)"
+PythonTurtleBackward = "x pixelt hátra"
+PythonTurtleCircle = "r pixel sugarú kört rajzolni"
+PythonTurtleColor = "Toll szinét beállitani"
+PythonTurtleColorMode = "Szin módot 1.0-ra vagy 255-ra állitani"
+PythonTurtleForward = "x pixelt elölre"
+PythonTurtleFunction = "turtle modul funkció elötag"
+PythonTurtleGoto = "Manjen a (x,y) koordinátákra"
```
Comment (Gegenter) [inline apps/code/catalog.hu.i18n:181]:
```suggestion
PythonTurtleGoto = "Menjen a (x,y) koordinátákra"

```

**Review by Gegenter — CHANGES_REQUESTED** [review]
Only 3 things to correct otherwise good job!!.

**Review by lapingenieur — COMMENTED** [review]
More hungarian changes thanks to @Gegenter

File: apps/code/catalog.hu.i18n [hunk 5]
```diff
@@ -36,187 +36,188 @@ PythonColorOrange = "Narancssárga szín"
...
+PythonTimeFunction = "time funkció elötag"
+PythonTrunc = "Egész csonka (?)"
+PythonTurtleBackward = "x pixelt hátra"
+PythonTurtleCircle = "r pixel sugarú kört rajzolni"
+PythonTurtleColor = "Toll szinét beállitani"
+PythonTurtleColorMode = "Szin módot 1.0-ra vagy 255-ra állitani"
+PythonTurtleForward = "x pixelt elölre"
+PythonTurtleFunction = "turtle modul funkció elötag"
+PythonTurtleGoto = "Menjen a (x,y) koordinátákra"
+
```
Comment (lapingenieur) [inline apps/code/catalog.hu.i18n:182]: Is this line that messes up the test ?

File: apps/code/base.hu.i18n [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
 Autocomplete = "Önkiegészítés"
 AutoImportScript = "Script automata importálása"
-BuiltinsAndKeywords = "Beépítet fonkciók és szókincs"
+BuiltinsAndKeywords = "Beépített fonkciók és szókincs"
 Console = "Konzol"
 DeleteScript = "Script törlése"
 DuplicateScript = "Script másolása"
 ExecuteScript = "Script indítása"
-FunctionsAndVariables = "Függvények és változók"
+FunctionsAndVariables = "Fonktiók és változók"
```
Comment (Gegenter) [inline apps/code/base.hu.i18n:10]: This is wrong, the other one was correct, "Függvények" is the correct term.

File: apps/on_boarding/base.hu.i18n [hunk 7]
```diff
@@ -1,7 +1,7 @@
 UpdateAvailable = "Elérhetö Frissítések!"
 UpdateMessage1 = "Fontos frissítések vannak"
 UpdateMessage2 = "a számológépéhez."
-UpdateMessage3 = "Látogasson el a weboldalunkra a számítógépröl"
+UpdateMessage3 = "Töltje laa weboldalunkról az utolso verziót"
```
Comment (Gegenter) [inline apps/on_boarding/base.hu.i18n:4]: Should be "Töltse le a weboldalunkról az utolsó verziót"

File: apps/regression/base.hu.i18n [hunk 8]
```diff
@@ -1,21 +1,20 @@
...
 RegressionCurve = "Regressziós görbe"
-XPrediction = "Jóslás megadva X"
+XPrediction = "Jóslás X megadva"
 YPrediction = "Jóslás Y megadva"
 ValueNotReachedByRegression = "Az ablakban az érték még nem volt elérve"
 NumberOfDots = "Pontok száma"
 Covariance = "Kovariancia"
 Linear = "Lineáris"
-Proportional = "Proportional"
+Proportional = "Általános"
```
Comment (Gegenter) [inline apps/regression/base.hu.i18n:12]: Proportional is technically "Arányos" in Hungarian altough when speaking about a functions and such it should be "egyenes arányosság" but i think "Arányos" is enough and everyone understands.

File: apps/regression/base.hu.i18n [hunk 9]
```diff
@@ -1,21 +1,20 @@
...
 Quadratic = "Másodfokú"
-Cubic = "Kocka"
+Cubic = "Kockás"
 Quartic = "Kvartikus"
 Logarithmic = "Logaritmikus"
 Power = "Teljesítmény"
 Trigonometrical = "Trigonometrikus"
 Logistic = "Logisztikai"
-DataNotSuitableForRegression = " Az adat nem megfelelö ehhez a regressziós modellhez"
+DataNotSuitableForRegression = "Azek az adatok nem megfelelö ehhez a regressziós modellhez"
```
Comment (Gegenter) [inline apps/regression/base.hu.i18n:20]: "Ezek az adatok nem megfelelőek ehhez a regressziós modellhez" <-- this is correct

File: apps/settings/base.hu.i18n [hunk 10]
```diff
@@ -12,21 +12,21 @@ ToDeactivateExamMode1 = "a vizsga mód kikapcsoláshoz"
...
+ExamModeWarning1 = "Vigyázat: a használt szoftver nem"
+ExamModeWarning2 = "hivatalos, Numworks nem garantálja"
+ExamModeWarning3 = "a vizsgálati mód megfelelőségét."
+AboutWarning1 = "Vigyázat: a használt szoftver"
+AboutWarning2 = "nem hivatalos. A NumWorks nem"
+AboutWarning3 = "vállal felelösséget az"
+AboutWarning4 = "esetleges károkért."
 # -----------------------------------------------------------------------------
-About = "Egyéb"
+About = "Apropó"
```
Comment (Gegenter) [inline apps/settings/base.hu.i18n:23]: "Egyéb" is more "professional" i would not use "apropó" here

File: apps/shared.hu.i18n [hunk 11]
```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
...
 ClearColumn = "Oszlop törlése"
 ColumnOptions = "Oszlop opciók"
-CopyColumnInList = "Az oszlop exportálása egy listába"
-DataNotSuitable = "Az adatok nem megfelelöek"
-DataTab = "Data"
-DefaultSetting = "Alapbeállítások"
+CopyColumnInList = "Az oszlopot egy listába másolni"
+DataNotSuitable = "Az adatok nem felelnek meg"
+DataTab = "Adatok"
+DefaultSetting = "Alapállítások"
```
Comment (Gegenter) [inline apps/shared.hu.i18n:17]: "Alapbeállítások" was good, i dunno why it was changed, maybe an error.

File: apps/shared.hu.i18n [hunk 12]
```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
...
 Deg = "deg"
 Deviation = "Varianca"
 DisplayValues = "Értékek mutatása"
 Empty = "Üres"
 Eng = "eng"
-ExitExamMode1 = "Kilépés a vizsga "
+ExitExamMode1 = "Kilépni a vizsga "
```
Comment (Gegenter) [inline apps/shared.hu.i18n:23]: "Kilépés" was the correct one gramatically,

File: apps/shared.hu.i18n [hunk 13]
```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
...
-HardwareTestLaunch4 = "az összes adat törlödni fog."
+HardwareTestLaunch1 = "A hardverteszt indítása :"
+HardwareTestLaunch2 = "Nyomjon a reset gombra a"
+HardwareTestLaunch3 = "teszt megállításához (ez"
+HardwareTestLaunch4 = "az adatokat törölni fogja)"
 Initialization = "Elöbeállítás"
 IntervalSet = "Állítsa be az intervallumot"
 Language = "Nyelv"
-LowBattery = "Alacsony töltöttség"
+LowBattery = "Majdnem kimerült az elem"
```
Comment (Gegenter) [inline apps/shared.hu.i18n:39]: "Alacsony töltöttség" is more professional and shorter, I think we should go with that.

File: apps/shared.hu.i18n [hunk 14]
```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
...
+RoundAbscissa = "Egész abszcissza"
 Sci = "sci"
 SquareSum = "Négyzetek összege"
 StandardDeviation = "Alap eltérés"
-StoreExpressionNotAllowed = "'tárolás' nem engedélyezett"
 StatTab = "Statisztikák"
 Step = "Lépés"
 StorageMemoryFull1 = "A memória megtelt."
-StorageMemoryFull2 = "Adatok törlése és próbálkozzon újra."
+StorageMemoryFull2 = "Törlöljön adatokat és próbálkozzon újra."
```
Comment (Gegenter) [inline apps/shared.hu.i18n:66]: "Töröljön" is the correct form.

File: apps/shared.hu.i18n [hunk 15]
```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
...
-UndefinedValue = "Nincs meghatározva érték"
+UndefinedValue = "Meghatározatlan adat"
 ValuesTab = "Táblázat"
 Warning = "Figyelem"
 XEnd = "X vége"
 XStart = "X kezdete"
-Zoom = "Zoom"
-Developers = "Fejlesztök"
+Zoom = "Nagyítás"
+Developers = "Kifejlesztök"
```
Comment (Gegenter) [inline apps/shared.hu.i18n:82]: Although we say "kifejlesztők" its generally meant for vaccines and such. "Fejlesztők" is the correct term here.

File: apps/solver/base.hu.i18n [hunk 16]
```diff
@@ -1,29 +1,29 @@
 SolverApp = "Egyenletek"
-SolverAppCapital = "Egyenletek"
-AddEquation = "Egyenlet hozzáadása"
-ResolveEquation = "Oldja meg az egyenletet"
-ResolveSystem = "A rendszer megoldása"
+SolverAppCapital = "EGYENLETEK"
+AddEquation = "Egyenlet hozzáadás"
```
Comment (Gegenter) [inline apps/solver/base.hu.i18n:3]: "hozzáadása"

File: apps/solver/base.hu.i18n [hunk 17]
```diff
@@ -1,29 +1,29 @@
...
+SolverAppCapital = "EGYENLETEK"
+AddEquation = "Egyenlet hozzáadás"
+ResolveEquation = "Egyenletet megoldása"
+ResolveSystem = "Rendszer megoldása"
 UseEquationModel = "Használjon egyenleti sablont"
-RequireEquation = "A bemenetnek egyenletnek kell lennie"
-UnrealEquation = "Unreal egyenlet"
-UndefinedEquation = "Definiálatlan egyenlet"
-TooManyVariables = "Túl sok ismeretlen"
+RequireEquation = "A bemenetnek csak egyenlet lehet"
```
Comment (Gegenter) [inline apps/solver/base.hu.i18n:7]: We can write "A bemenet csak egyenlet lehet" or " A bemenetnek egyenletnek kell lennie" both are correct.

File: apps/solver/base.hu.i18n [hunk 18]
```diff
@@ -1,29 +1,29 @@
...
-NoSolutionInterval = "Nem találtunk megoldást az intervallumban"
-EnterEquation = "Írja be az egyenletet"
-InfiniteNumberOfSolutions = "Végtelen számú megoldás létezik"
-ApproximateSolutionIntervalInstruction0 = "Adja meg a keresés intervallumát"
-ApproximateSolutionIntervalInstruction1 = "egy hozzávetöleges megoldáshoz"
-OnlyFirstSolutionsDisplayed0 = "Csak az elsö 10 megoldás"
-OnlyFirstSolutionsDisplayed1 = "megjelenik"
+NoSolutionInterval = "Nincs megoldás ebben az intervallumban"
+EnterEquation = "Írjon be egy egyenletet"
+InfiniteNumberOfSolutions = "Végtelen menyi megoldások léteznek"
```
Comment (Gegenter) [inline apps/solver/base.hu.i18n:19]: Once again the other one was good, the changed one makes no sense grammatically. We should use " Végtelen számú megoldás létezik"

File: apps/solver/base.hu.i18n [hunk 19]
```diff
@@ -1,29 +1,29 @@
...
+ApproximateSolutionIntervalInstruction0 = "Irja be a keresés intervallumot"
+ApproximateSolutionIntervalInstruction1 = "amelyikbe egy hozzávetöleges megoldást keressen"
+OnlyFirstSolutionsDisplayed0 = "Csak a 10 elsö megoldások"
```
Comment (Gegenter) [inline apps/solver/base.hu.i18n:22]: "Csak az első 10 megoldás"

File: apps/solver/base.hu.i18n [hunk 20]
```diff
@@ -1,29 +1,29 @@
...
+OnlyFirstSolutionsDisplayed1 = "jelennek meg"
```
Comment (Gegenter) [inline apps/solver/base.hu.i18n:23]: "Jelenjen meg"

File: apps/variables.hu.i18n [hunk 21]
```diff
@@ -1,10 +1,10 @@
 Variables = "Változók"
 Expressions = "Kifejezések"
-Functions = "Függvények"
-EmptyExpressionBox0 = "Nem határozott meg egyetlen változót sem."
-EmptyFunctionBox0 = "Nem határozott meg egyetlen függvényt sem."
+Functions = "Funciók"
```
Comment (Gegenter) [inline apps/variables.hu.i18n:3]: Függvény is correct "Funciók" doesnt exist.

File: apps/variables.hu.i18n [hunk 22]
```diff
@@ -1,10 +1,10 @@
 Variables = "Változók"
 Expressions = "Kifejezések"
-Functions = "Függvények"
-EmptyExpressionBox0 = "Nem határozott meg egyetlen változót sem."
-EmptyFunctionBox0 = "Nem határozott meg egyetlen függvényt sem."
+Functions = "Funciók"
+EmptyExpressionBox0 = "Nincs meghatározott változó."
+EmptyFunctionBox0 = "Nincs meghatározott funkció."
```
Comment (Gegenter) [inline apps/variables.hu.i18n:5]: Although grammatically correct, i would change funkció to függvény.

File: apps/variables.hu.i18n [hunk 23]
```diff
@@ -1,10 +1,10 @@
...
-EmptyExpressionBox0 = "Nem határozott meg egyetlen változót sem."
-EmptyFunctionBox0 = "Nem határozott meg egyetlen függvényt sem."
+Functions = "Funciók"
+EmptyExpressionBox0 = "Nincs meghatározott változó."
+EmptyFunctionBox0 = "Nincs meghatározott funkció."
 EmptyExpressionBox1 = "A változó meghatározásához írja be:"
-EmptyFunctionBox1 = "Függvény meghatározásához írja be:"
-EmptyExpressionBox2 = "A változó neve tartalmazhatja:"
-EmptyFunctionBox2 = "A függvény neve tartalmazhatja:"
+EmptyFunctionBox1 = "Funkció meghatározásához írja be:"
```
Comment (Gegenter) [inline apps/variables.hu.i18n:7]: Függvény

File: apps/variables.hu.i18n [hunk 24]
```diff
@@ -1,10 +1,10 @@
...
+EmptyExpressionBox2 = "A változó neve tartalmazhat:"
+EmptyFunctionBox2 = "A funkció neve tartalmazhat:"
```
Comment (Gegenter) [inline apps/variables.hu.i18n:9]: Függvény

**Review by Gegenter — CHANGES_REQUESTED** [review]
That's about it, good job, only a couple of errors 👍 

-Gegenter

**Review by lapingenieur — COMMENTED** [review]
If I did right @Gegenter 's changes should be added to this pr

**Pull request merged**